### PR TITLE
Update Afterpay API version to V2

### PR DESCRIPTION
**Pull request by jyucis**
New API version allows for a traditional auth then capture
model of handling payments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,44 +84,23 @@ Getting Started
...
+See [https://developer.workarea.com](https://developer.workarea.com) for Workarea Commerce documentation.
 
-Copyright & Licensing
+License
 --------------------------------------------------------------------------------
 
-Copyright WebLinc 2018. All rights reserved.
-
-For licensing, contact sales@workarea.com.
+Workarea Blog is released under the [Business Software License](LICENSE)
```
Comment (eric-pigeon) [inline README.md]: Blog -> Afterpay

**Review by eric-pigeon — CHANGES_REQUESTED** [review]

**Review by meowsus — APPROVED** [review]

**Comment by meowsus** [comment]
I think this is good to be merged @jyucis

**Comment by bencrouse** [comment]
What are we doing with this @eric-pigeon @jyucis ?

**Comment by jyucis** [comment]
I'd like to get @eric-pigeon to take a look before we merge.

**Review by eric-pigeon — CHANGES_REQUESTED** [review]
You aren't handling timeout issues on any external HTTP interactions with afterpay.  From the afterpay documentation the authorize, and capture actions are idempotent.  We should at the very least rescue any http timeout errors and retry them at least once, then properly fail and persisted a failed transaction.

**Comment by bencrouse** [comment]
@jyucis can you wrap this up? Let's try to finish work in progress.

File: README.md [hunk 2]
```diff
@@ -88,44 +82,23 @@ Getting Started
...
+    gem 'workarea-afterpay'
     # ...
 
 Update your application's bundle.
 
     cd path/to/application
     bundle
 
-Workarea Platform Documentation
+WebLinc Platform Documentation
```
Comment (eric-pigeon) [inline README.md]: Workarea?

**Comment by meowsus** [comment]
@jyucis @eric-pigeon I'm assuming this work is for the next patch: v2.0.3

Let me know if my assumption is far from correct.

**Comment by eric-pigeon** [comment]
I think this work for a new major since we updating to a new major for the afterpay API

**Comment by bencrouse** [comment]
@eric-pigeon where are we at with your review? Would you take a look?

**Review by eric-pigeon — CHANGES_REQUESTED** [review]

**Pull request merged**